.github/workflows/mend-cli-scan.yaml

```diff
@@ -49,45 +41,118 @@ jobs:
       - if: ${{ steps.cache-mend.outputs.cache-hit != 'true' }}
         name: Download Mend CLI executable (cache miss...)
         continue-on-error: true
+        shell: bash
```

Comment:
Even though GitHub defaults to `bash` on Ubuntu runners, it's safer to be explicit for portability...